lib/src/includes/signed_video_sign.h

```diff
@@ -118,6 +118,37 @@ signed_video_add_nalu_for_signing(signed_video_t *self,
     const uint8_t *nalu_data,
     size_t nalu_data_size);
 
+/**
+ * @brief Updates Signed Video, with a H26x NALU, for signing with timestamp
+ *
+ * For information regarding the function please reference the signed_video_add_nalu_for_signing.
```

Comment:
Since this API is an extension of `signed_video_add_nalu_for_signing()` we can use one description for both to minimize duplicated code.